sdk/websites/Azure.ResourceManager.AppService/api/Azure.ResourceManager.AppService.netstandard2.0.cs

```diff
@@ -4528,30 +4635,82 @@ public partial class AppServiceDomainPatch : Azure.ResourceManager.Models.Resour
+        public System.Collections.Generic.IReadOnlyList<Azure.ResourceManager.AppService.Models.AppServiceHostName> ManagedHostNames { get { throw null; } }
         public System.Collections.Generic.IReadOnlyList<string> NameServers { get { throw null; } }
         public Azure.ResourceManager.AppService.Models.ProvisioningState? ProvisioningState { get { throw null; } }
-        public Azure.ResourceManager.AppService.Models.DomainStatus? RegistrationStatus { get { throw null; } }
-        public Azure.ResourceManager.AppService.Models.DnsType? TargetDnsType { get { throw null; } set { } }
+        public Azure.ResourceManager.AppService.Models.AppServiceDomainStatus? RegistrationStatus { get { throw null; } }
+        public Azure.ResourceManager.AppService.Models.AppServiceDnsType? TargetDnsType { get { throw null; } set { } }
+    }
+    public enum AppServiceDomainStatus
+    {
+        Unknown = 0,
+        Active = 1,
+        Awaiting = 2,
+        Cancelled = 3,
+        Confiscated = 4,
+        Disabled = 5,
+        Excluded = 6,
+        Expired = 7,
+        Failed = 8,
+        Held = 9,
+        Locked = 10,
+        Parked = 11,
+        Pending = 12,
+        Reserved = 13,
+        Reverted = 14,
+        Suspended = 15,
+        Transferred = 16,
+        Unlocked = 17,
+        Unparked = 18,
+        Updated = 19,
+        JsonConverterFailed = 20,
+    }
+    public enum AppServiceDomainType
+    {
+        Regular = 0,
+        SoftDeleted = 1,
+    }
+    public partial class AppServiceEndpointDependency
+    {
+        internal AppServiceEndpointDependency() { }
+        public string DomainName { get { throw null; } }
+        public System.Collections.Generic.IReadOnlyList<Azure.ResourceManager.AppService.Models.AppServiceEndpointDetail> EndpointDetails { get { throw null; } }
+    }
+    public partial class AppServiceEndpointDetail
+    {
+        internal AppServiceEndpointDetail() { }
+        public System.Net.IPAddress IPAddress { get { throw null; } }
+        public bool? IsAccessible { get { throw null; } }
+        public double? Latency { get { throw null; } }
+        public int? Port { get { throw null; } }
+    }
+    public partial class AppServiceEnvironmentAddressResult : Azure.ResourceManager.Models.ResourceData
+    {
+        public AppServiceEnvironmentAddressResult() { }
+        public System.Net.IPAddress InternalIPAddress { get { throw null; } set { } }
+        public string Kind { get { throw null; } set { } }
+        public System.Collections.Generic.IList<System.Net.IPAddress> OutboundIPAddresses { get { throw null; } }
+        public System.Net.IPAddress ServiceIPAddress { get { throw null; } set { } }
+        public System.Collections.Generic.IList<Azure.ResourceManager.AppService.Models.VirtualIPMapping> VipMappings { get { throw null; } }
```

Comment:
The property name prefix `Vip` is misleading, it should be `VIPMappings` or consider expand it to `VirtualIPMappings`.

Reply:
fixed